Jenkinsfile

```diff
@@ -6,130 +6,85 @@ pipeline {
       choices: ['QUICK', 'FULL'],
       description: 'Build types:<p>QUICK: Compile, Build, Deploy artifacts, Skip integration tests and extra artifacts, Multithread build<p>FULL: Compile, Build, Deploy artifacts, Run integration tests and extra artifacts, Singlethread build\n'
     )
-    booleanParam(name: 'RELEASE', defaultValue: false, description: 'Make a Maven release')
```

Comment:
don't we need this to release the project?

Reply:
yes. fixed.